common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -92,7 +92,7 @@ public class CommonValue {
             + "\" : {\"type\": \"integer\"},\n"
             + "      \""
             + MLModel.PLANNING_WORKER_NODES_FIELD
-            + "\": {\"type\": \"keyword\"},\n"
+            + "\": {\"type\": \"text\"},\n"
```

Comment:
Why change to text?

Reply:
I remember the planningWorkerNodes is a string with comma as splitter, so I changed this, but I checked again and found it's already a list type, changing this back.